src/Faker/Provider/en_US/PhoneNumber.php

```diff
@@ -69,6 +70,15 @@ public function tollFreePhoneNumber()
         return self::numerify($this->generator->parse($format));
     }
 
+    /**
+     * @return string
+     * @example '555-123-546 x123'
+     */
+    public function phoneNumberWithExtension(): string
```

Comment:
Are we allowing return types in 1.x?

Reply:
Since this is a new method, not breaking existing code (except for if someone extends this class that includes a method with the exact same name and different return type), I don't think it is an issue.

Reply:
Yes, but it does lead to inconsistent code.

Reply:
Let's keep return types out of `1.x`.

Reply:
I've added it here because of the discussion on https://github.com/FakerPHP/Faker/pull/88#discussion_r532556814 - so I'll remove the return type there too.

Reply:
I should have been more consistent.